pycontw/pycontw-2021
# style: update Sponsorship and prospectus page

## Types of Changes

**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies.


## Description

Adjust the styles of Sponsorship and prospectus page to accord with design

## Steps to Test This Pull Request

Steps to reproduce the behavior:
1. Run dev server and json server
2. Go to sponsor page
3. Check styles
4. Click prospectus link
5. Check styles

## Expected Behavior

Correspondent styles with design

## More Information

**Screenshots**
<img width="1372" alt="截圖 2021-07-12 下午1 18 16" src="https://user-images.githubusercontent.com/13831865/125234606-9fcf9f80-e313-11eb-817d-bca4d4c8a81d.png">
<img width="1373" alt="截圖 2021-07-12 下午1 18 38" src="https://user-images.githubusercontent.com/13831865/125234644-ad852500-e313-11eb-9151-fdf4fe932808.png">
